jiant/scripts/download_data/datasets/nlp_tasks.py

```diff
@@ -110,9 +111,9 @@
     "superglue_winogender_diagnostics": {
         "path": "super_glue",
         "name": "axg",
-        "label_map": RteTask.ID_TO_LABEL,
+        "label_map": SuperglueWinogenderDiagnosticsTask.ID_TO_LABEL,
         "phase_map": None,
-        "jiant_task_name": "rte",
+        "jiant_task_name": "axg",
```

Comment:
Mind changing this to `superglue_axg`?

Reply:
resolved in d5172b9b0ade873782966f105c920c1a7ba2247d